assessments/0009-iin-toto.md

```diff
@@ -0,0 +1,292 @@
+
+- The  [contributing page](https://github.com/in-toto/community/blob/main/CONTRIBUTING.md) should explicitly mention documentation changes as distinct from "making code changes" as an area of contribution. 
+
+- The  [contributing page](https://github.com/in-toto/community/blob/main/CONTRIBUTING.md) needs explicit information for how to submit issues and feature requests for documentation, since documentation does not have a single "corresponding GitHub repository".
+
+
+## Website
+
+| Criteria                                    | 1   | 2   | 3   | 4   | 5   |
+| ---                                         | --- | --- | --- | --- | --- |
+| Single-source for all files                 |     |     |     |  x  |     |
+| Meets min website req. (for maturity level) |     |  x  |     |     |     |
+| Branding and design                         |     |     |     |     |     |
+| Case studies/social proof                   |     |     |  x  |     |     |
+| Maintenance planning                        |     |     |     |     |     |
+| A11y plan & implementation                  |     |     |     |     |     |
+| Mobile-first plan & impl.                   |     |     |     |     |     |
+| HTTPS access & HTTP redirect                |     |     |     |     | x   |
+| Google Analytics 4 for production only      |     |     |     |     |     |
+| Indexing allowed for production server only |     |     |     |     |     |
+| Intra-site / local search                   |     |  x  |     |     |     |
+| Account custodians are documented           |     |     |     |     |     |
+
+Scale:
+
+- 1 = (Is not present or requires significant work)
+- 3 = (Is present, but needs work)
+- 5 = (Is executed extremely well or no improvement required)
+
+**Comments**
+
+- **Single-source for all files:** Source is in [in-toto.io repo](https://github.com/in-toto/in-toto.io/blob/master/content/in-toto.md). 
+
+- **Meets min website req. (for Incubating):** Doc assessment in progress. Very little documentation is published directly on the website. Most docs are in GitHub, only a few of those are directly linked from the website.
+
+- **Case studies/social proof:** Menu item **About > Adoptions & Integrations** links to GitHub repo with folders created and maintained by adopters. No icons or listing of adopters on the web site.
+
+- **HTTPS access & HTTP redirect:** HTTPS is the default, HTTP redirects correctly.
+
+- **Intra-site / local search:** No site map or search mechanism, only navigation is through a minimal menu bar.
+
+_Provide comments for each rating above, 1-2 sentences max, bullet point list_
+_Include a list of the top 404s, as reported through analytics or a search console._
+
+**Recommendations**
+
+_Provide a list of recommendations to improve in this area_
```

Comment:
Let me take a closer look at that PR and I'll circle around back to this one. On first glance, it looks like a good idea.